mmv1/templates/terraform/resource.html.markdown.erb

```diff
@@ -100,7 +100,7 @@ state as plain-text. [Read more about sensitive data in state](/docs/state/sensi
      Unless/Until we split our docs, this is a non-issue. -%>
 <% unless object.examples.empty? -%>
   <%- object.examples.reject(&:skip_docs).each do |example| -%>
-    <%- unless example.skip_test -%>
+    <%- unless example.skip_test || !example.test_env_vars.nil? -%>
```

Comment:
Should we apply the same `&& e.test_env_vars.any?` as in https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/provider/terraform_oics.rb#L37?

Reply:
Not sure if there's any cases where it comes up, but seems prudent to match exactly

Reply:
shouldn't hurt!